src/ducks/settings/ToggleRow.jsx

```diff
@@ -31,7 +31,7 @@ class ToggleRow extends Component {
             {hasValue && (
               <input
                 type="number"
-                onChange={e => onChangeValue(name, parseNumber(e.target.value))}
+                onChange={e => onChangeValue(parseNumber(e.target.value))}
```

Comment:
As we used a function here the ToggleRow always rerender even if Props don't change. Same line 49